.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "go",
+            "request": "launch",
+            "name": "Debug App",
+            "program": "${workspaceFolder}/main.go",
+            "args": [
+                "--api-export-name", "build-service-apiexport",
+                "--zap-log-level", "6",
+                "--zap-devel", "true",
+            ],
+            "env": {
+                "KUBECONFIG":"/home/user/projects/kcp/.kcp/admin.kubeconfig"
```

Comment:
this is too much specific so the file should not be included in the commit.

Reply:
ok, will remove it